admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -25,16 +26,36 @@ Or, passwords are not stored and available only for the current session, which a
 Installation
 ------------
 
-Enable the Windows Network Drive app on your ownCloud Apps page. 
-Then there are a few dependencies to install.
+Install `the External Storage: Windows Network Drives app`_ from the ownCloud Market App or ownCloud Marketplace. 
```

Comment:
Why "Dirves", as far as I know the app is called "Windows Network Drive"

Reply:
The name I used was taken from the app's Marketplace URL.